src-tauri/src/main.rs

```diff
@@ -151,8 +175,23 @@ pub(crate) async fn farm(
 
         move || Plot::open_or_create(&base_directory)
     });
+
     let plot = plot_fut.await.unwrap()?;
 
+    let mut _handler_guard = HANDLER
```

Comment:
I don't understand why you have `let mut _handler_guard =` and why you need to store it at all. Maybe you can just call `.detach()` on handler instead?

Reply:
Why do you need to store handler at all?

Reply:
I don't. It was there because of the old trials. I don't store it, this is changed with the last commit

Reply:
I thought you were referring to `let mut _handler_guard`. 
So I need a `HANDLER` to be able to keep track of updates on the plotting pieces. 
This needs to be initiated in `farm` function. However, `farm` function returns quickly, and because of that the `HANDLER` needs to be a global variable to outlive the farmer. At least, that was the solution I came up with.

Reply:
I was referring both to that (because it wasn't even a guard) and the fact that you store anything in the first place. I also mentioned method that you can call instead of storing it and achieve the same result with less dependencies and less code.

Reply:
I think it is fine now :)